sys-libs/mesa/mesa-17.1.7.recipe

```diff
@@ -149,24 +137,3 @@ EOF
 		"$addOnsDir/opengl/Software Pipe"
 }
 
-
```

Comment:
Wait.. why are we dropping the renderer package? We want more of these going forward... not to remove them.

Reply:
The definition is just moved before BUILD_REQUIRES, like for every other recipe.